pkg/graveler/committed/iterator.go

```diff
@@ -21,12 +21,13 @@ type iterator struct {
 	beforeRange bool
 }
 
-func NewIterator(ctx context.Context, manager RangeManager, namespace Namespace, rangesIt ValueIterator) Iterator {
+func NewIterator(ctx context.Context, manager RangeManager, storageID StorageID, namespace Namespace, rangesIt ValueIterator) Iterator {
 	return &iterator{
 		ctx:       ctx,
 		manager:   manager,
 		namespace: namespace,
 		rangesIt:  rangesIt,
+		storageID: storageID,
```

Comment:
According to the convention, should be placed just before namespace.

Reply:
It's a struct intialization code - I find this irrelevant